octodns/provider/route53.py

```diff
@@ -674,10 +675,16 @@ def _get_zone_id(self, name, create=False):
             return id
         if create:
             ref = uuid4().hex
+            del_set = self.delegation_set_id
             self.log.debug('_get_zone_id:   no matching zone, creating, '
                            'ref=%s', ref)
-            resp = self._conn.create_hosted_zone(Name=name,
-                                                 CallerReference=ref)
+            if self.delegation_set_id:
+                resp = self._conn.create_hosted_zone(Name=name,
+                                                     CallerReference=ref,
+                                                     DelegationSetId=del_set)
```

Comment:
Can `DelegationSetId=None` be passed here to avoid the `if`?

Reply:
no i've tested it and it failed

Reply:
:-(, that's actually why I was asking, wouldn't really have to worry about test coverage in that case.